File: src/linkerz_crawler_core/src/main/java/org/linkerz/crawler/core/model/WebLink.java
/*
 * Copyright (C) 2012 - 2013 LinkerZ
 */

package org.linkerz.crawler.core.model;

import java.io.Serializable;

/**
 * The Class WebLink.
 *
 * This class represent for the web link.
 *
 * @author Nguyen Duc Dung
 * @since 7/2/12, 1:02 AM
 */
public class WebLink implements Serializable {

    private String url;

    public WebLink(String url) {
        this.url = url;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}

File: src/linkerz_crawler_core/src/main/scala/org/linkerz/crawler/core/model/WebLink.scala
package org.linkerz.crawler.core.model

/**
 * The Class WebLink.
 *
 * @author Nguyen Duc Dung
 * @since 7/2/12, 11:39 PM
 *
 */

class WebLink {

  var _url: String = _

  def this(url: String) {
    this()
    _url = url
  }

  def url = _url

  def url_=(url: String) {
    _url = url
  }

}
